packages/styles/core-global.css

```diff
@@ -10,3268 +10,9 @@ OF ANY KIND, either express or implied. See the License for the specific languag
-        --spectrum-global-color-static-indigo-700
-    );
-    --spectrum-alias-categorical-color-3: var(
-        --spectrum-global-color-static-orange-500
-    );
-    --spectrum-alias-categorical-color-4: var(
-        --spectrum-global-color-static-magenta-500
-    );
-    --spectrum-alias-categorical-color-5: var(
-        --spectrum-global-color-static-indigo-200
-    );
-    --spectrum-alias-categorical-color-6: var(
-        --spectrum-global-color-static-celery-200
-    );
-    --spectrum-alias-categorical-color-7: var(
-        --spectrum-global-color-static-blue-500
-    );
-    --spectrum-alias-categorical-color-8: var(
-        --spectrum-global-color-static-purple-800
-    );
-    --spectrum-alias-categorical-color-9: var(
-        --spectrum-global-color-static-yellow-500
-    );
-    --spectrum-alias-categorical-color-10: var(
-        --spectrum-global-color-static-orange-700
-    );
-    --spectrum-alias-categorical-color-11: var(
-        --spectrum-global-color-static-green-600
-    );
-    --spectrum-alias-categorical-color-12: var(
-        --spectrum-global-color-static-chartreuse-300
-    );
-    --spectrum-alias-categorical-color-13: var(
-        --spectrum-global-color-static-blue-200
-    );
-    --spectrum-alias-categorical-color-14: var(
-        --spectrum-global-color-static-fuchsia-500
-    );
-    --spectrum-alias-categorical-color-15: var(
-        --spectrum-global-color-static-magenta-200
-    );
-    --spectrum-alias-categorical-color-16: var(
-        --spectrum-global-color-static-yellow-200
-    );
-}
 :host,
 :root {
-    --spectrum-colorloupe-express-visibility: none;
-    --spectrum-colorloupe-spectrum-visibility: block;
-    --spectrum-colorloupe-outer-border-color: transparent;
-    --spectrum-colorloupe-outer-border-size: 0;
-    --spectrum-colorloupe-outer-stroke-color: var(
-        --spectrum-global-color-static-transparent-black-200
-    );
-    --spectrum-colorloupe-outer-stroke-width: var(
-        --spectrum-alias-border-size-thick
-    );
-    --spectrum-colorhandle-background-offset: calc(
-        var(--spectrum-global-dimension-static-size-25) * -1
-    );
-    --spectrum-colorhandle-inner-shadow-color: var(
-        --spectrum-colorhandle-outer-shadow-color
-    );
-    --spectrum-colorhandle-outer-shadow-color: rgba(0, 0, 0, 0.42);
-    --spectrum-colorhandle-outer-shadow-blur: 0;
-    --spectrum-colorhandle-outer-shadow-spread: var(
-        --spectrum-alias-border-size-thin
-    );
-    --spectrum-colorcontrol-checkerboard-light-color: var(
-        --spectrum-global-color-static-white
-    );
-    --spectrum-colorcontrol-checkerboard-dark-color: var(
-        --spectrum-global-color-static-gray-300
-    );
-    --spectrum-slider-m-track-inside-border-radius: var(
-        --spectrum-slider-m-track-border-radius
-    );
-    --spectrum-slider-label-text-size: var(
-        --spectrum-global-dimension-font-size-75
-    );
+    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
```

Comment:
Pretty ugly in the diff, but here's the magic sauce. Could conceivably centralize this to a single file the maybe made this more clear?

Reply:
Thanks, no action required, just curiosity.